### DISCUSSION TITLE
页面组件多的话可以自动合并加载

### DISCUSSION THREAD

**Discussion by LogicHou** [ofajs/discussion#6] [Q&A] [answered]
我是搞后端的，不是太懂前端，就比如我一个页面有很多组件的时候（比如10个），就需要加载10个组件文件吗？框架能自动合并一部分文件吗，还是要自己手动合并到一个里面

**Comment by kirakiray** [comment, marked as answer]
目前有[工具](https://builder.ofajs.com/)可以压缩混淆文件，暂时还不支持合并成一个文件。如果这10个组件是在同一个域名下，并且是http2的服务器下，请求数会有优化。

这个合并成一个文件的需求紧急吗，如果是你已经使用了ofa.js，并且上头有强制指标需要打包成一个文件，我这里可以安排一下将多个组件合并的任务提前。

**Reply by LogicHou** [reply]
没有那么紧急，就是好奇问一下。我是想自己做一个小站，还在研究ofa.js的阶段，您可以按照自己的todo来，有需求我再来麻烦您

**Comment by kirakiray** [comment]
> 没有那么紧急，就是好奇问一下。我是想自己做一个小站，还在研究ofa.js的阶段，您可以按照自己的todo来，有需求我再来麻烦您

好的，有情况可以随时反馈。
